prep-html-skeleton/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+  <head></head>
+    <title>HTML Skeleton Exercise </title>
```

Comment:
Hi Nick, try structuring the document like the example in the linked resources. Specifically the part under HTML Page Structure.

https://www.w3schools.com/html/html_intro.asp